### DISCUSSION TITLE
Maven 3.9.x release after Maven 4.0.0 has been published

### DISCUSSION THREAD

**Discussion by bmarwell** [support-and-care/maven-support-and-care#16] [Ideas for Funding]
Since volunteers will probably not work on the 3.x branch anymore after there was a final/GA Maven 4 release, we could create a 3.x maintenance release in case more bugs are found which need to be addressed for 3.9 users.

## Goals

* Collect the most voted issues for 3.9.x on the Apache issue tracker
* Collect all the blockers
* (Optional) See whether a customer wishes to have a release and eventually add a few of their wish-list issues (only defects)
* Encourage users to migrate to 4.x (maybe add a warning which can be deactivated using an env var)

## Non-Goals

* create a 3.8.x release or lower
* Create a backport of an important Maven 4 feature
* Create a new feature not present in Maven 4 (unless it helps migration to Maven 4, to be discussed)
* Encourage users to stick with 3.x

**Comment by hendrikebbers** [comment]
Some questions:

- Do we now how often a critical bugfix should be ported back to Maven 3?
- Will we have Maven community reviewers willing to review PRs for old branches (Maven 3)?
- If we do that, we could only support a Maven 3 branch for one year (based on funding). Based on that, the milestone needs to contain the goal of defining an idea of how LTS could even be continued in the future.

**Reply by bmarwell** [reply]
* How often: I (personally) think we should decide by severity. If less-severe, collect more changes for a release.
* Reviewers: Probably yes, I honestly do not think this will be an issue. 

The LTS part / continued support would need to be decided by the community. We need to put that on the mailing list.

**Reply by rmannibucau** [reply]
@bmarwell the discussion already happent (multiple times AFAIK) and the decision (not mine if ambiguous) was no engagement from the community and let it be at will so don't think we need any new discussion. If there is a need just push for it I'd say.

**Reply by bmarwell** [reply]
Yes, No Engagement in fixing. But a quick review should be possible, no? I'd be willing to do this, for example.

**Reply by rmannibucau** [reply]
See it as there is no blocker nor not any will to do it  from my understanding so if somebody wants to do it is possible but there will be no engagement of the community about it.

**Comment by ascheman** [comment]
This discussion has been converted to Epic #41 for tracking.

Closing this discussion.
